### DISCUSSION TITLE
Version 4 vs 5 beta ?

### DISCUSSION THREAD

**Discussion by MateuszLas421** [Mapsui/Mapsui#2661] [Q&A] [answered]
I'm currently migrating from version 3.X.X. My question is, does 5.0.0-beta1 include the latest fixes from version 4.1.6? If there are any updates in version 4.X.X, will the 5.0.0 betaX version be updated accordingly? Or is it better to stick with version 4 for now?

**Comment by pauldendulk** [comment, marked as answer]
I think it is better to stick to version 4. More changes will follow in v5 and we will have introduced bugs. Most of the fixes of v4 are in v5. I think only the Blazor mouse wheel sensitivity fix is not yet there.
